modules/RTC/RTCUtils.js

```diff
@@ -709,8 +710,11 @@ function handleLocalStream(streams, resolution) {
                 videoStream = new MediaStream();
                 for (let j = 0; j < videoTracks.length; j++) {
                     videoStream.addTrack(videoTracks[j]);
+                    audioVideo.removeTrack(videoTracks[j]);
                 }
             }
+
+            audioVideo.release();
```

Comment:
This won't work on web. We can check for it like so.
```suggestion
            audioVideo.release && audioVideo.release();
```